website/docs/r/lb_server_pool.html.markdown

```diff
@@ -0,0 +1,129 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_lb_server_pool"
+sidebar_current: "docs-vcd-resource-lb-server-pool"
+description: |-
+  Provides a NSX load balancer server pool data source.
+---
+
+# vcd\_lb\_server\_pool
+
+Provides a vCloud Director Edge Gateway Load Balancer Server Pool data source. A Server Pool defines
+a group of backend servers (defined as pool members), manages load balancer distribution methods, and has a service 
+monitor attached to it for health check parameters.
+
+
+~> **Note:** To make load balancing work one must ensure that load balancing is enabled on edge gateway. This depends 
+on NSX version to work properly. Please refer to [VMware Product Interoperability Matrices](https://www.vmware.com/resources/compatibility/sim/interop_matrix.php#interop&29=&93=) 
+to check supported vCloud director and NSX for vSphere configurations.
+
+~> **Note:** The vCloud Director API for NSX supports a subset of the operations and objects defined in the NSX vSphere 
+API Guide. The API supports NSX 6.2, 6.3, and 6.4.
+
+Supported in provider *v2.4+*
+
+## Example Usage
+
+```hcl
+provider "vcd" {
+  user     = "${var.admin_user}"
+  password = "${var.admin_password}"
+  org      = "System"
+  url      = "https://AcmeVcd/api"
+}
+
+resource "vcd_lb_server_pool" "web-servers" {
+  org                 = "my-org"
+  vdc                 = "my-org-vdc"
+  edge_gateway        = "my-edge-gw"
+
+  name                = "web-servers"
+  description         = "description"
+  algorithm           = "round-robin"
+
+  monitor_id = "${vcd_lb_service_monitor.lb-service-monitor.id}"
```

Comment:
OK, I believe we'll get to it eventually, but we'll need one super sample somewhere where all the pieces connect up :)

Reply:
Added a datasource usage for service monitor resource reference and put a link in the service monitor datasource documentation.